pkg/storage/minio/cataloger_test.go

```diff
@@ -0,0 +1,40 @@
+package minio
+
+import (
+	"context"
+	"testing"
+
+	"github.com/gomods/athens/pkg/paths"
+	"github.com/gomods/athens/pkg/storage/minio/mocks"
+	"github.com/minio/minio-go/v6"
+	"github.com/stretchr/testify/require"
+)
+
+func TestCatalog(t *testing.T) {
```

Comment:
Instead of mocks can we use a real minio set up like here

https://github.com/gomods/athens/blob/main/pkg/storage/minio/minio_test.go